### PR TITLE
Reopening of #22332 with some fun tweaks

### PR DESCRIPTION
**Pull request by ma44**
:cl: ma44
add: Meatwheat now has a smoke gene
add: Steelcaps now produce 5u of iron at max potency
tweak: Nettle has half the sacid (50u at 100 potency to 25) Deathnettle is unaffected
tweak: Corn has more cornoil, from 20 units at 100 potency to 30 units
/:cl:

Tweaks are less controversial, iron is now lowered by a lot, so you can't make 2,4-5 emps instantly. You can still grind up all the iron from the steelcap to actually get a decent amount of iron.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
I reallyyyy don't think botany needs more buffs right now, to be honest...and some genes that have been added in should be fairly difficult to get---particularly the smoke one.

**Comment by jughu** [comment]
>get novaflowers nd bracr my teargas smoke plants!

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
Please modify the title of your pr to clearly state what it does.

**Comment by kevinz000** [comment]
What @Fox-McCloud said. Botany is powerful enough, a good botanist can take literally anything that isn't a mech down and get to near old-viro healing rates and stun immunity. Their kudzu can be more powerful than a blob. They don't need to be able to produce emps and get smoke plants that instakill rooms of people too.

**Comment by jughu** [comment]
WHY DO PEOPLE WANT MEATWHEAT SMOKE GENE SO BADLY

**Comment by jughu** [comment]
WHAT ISS THIS MEME WITH NERFING NORMAL NETTLES BUFFING CORN

File: code/modules/hydroponics/grown/nettle.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	growthstages = 5
 	genes = list(/datum/plant_gene/trait/repeated_harvest, /datum/plant_gene/trait/plant_type/weed_hardy)
 	mutatelist = list(/obj/item/seeds/nettle/death)
-	reagents_add = list("sacid" = 0.5)
+	reagents_add = list("sacid" = 0.25)
```
Comment (jughu) [inline code/modules/hydroponics/grown/nettle.dm]: Why???

**Review by jughu — CHANGES_REQUESTED** [review]
Why do you even want to nerf this what is this reasoniiing

**Comment by vcordie** [comment]
a, you fucked up the ratio on regular nettles. this will still leave a lot of leftover acid and not fix qol.
b. don't copy my earlier prs damn it. if you wanted to reopen the discussion on the iron trait, just open a pr with your new tweaked iron trait, instead of also making my earlier prs that I've now put up redundant.

**Comment by jughu** [comment]
So theres like 3 prs open for the same fucking thing what is this meme and there isnt even a reasoning

**Comment by ChangelingRain** [comment]
Please nerf botany before adding more buffs to it.

**Pull request closed without merging**